src/Okta.Idx.Sdk.UnitTests/IdxClientShould.cs

```diff
@@ -1013,5 +1013,50 @@ public async Task CallInteractOnStartSignInWidget()
             Assert.Equal("this is a test interaction handle", widgetResponse.IdxContext.InteractionHandle);
             Assert.Equal(1, mockHttpMessageHandler.CallCounts["/v1/interact"]);
         }
+
+        [Fact]
+        public async Task ProcessInfoMessageResponse()
```

Comment:
I suggest we keep all "parsing" UTs in the `IdxClientLayer1Should` class and use this one for the "layer 3" tests (public IdxClient methods).